orca-sql/src/main/resources/db/changelog/20190604-stages-updatedat-index.yml

```diff
@@ -0,0 +1,24 @@
+databaseChangeLog:
+  - changeSet:
+      id: 20190604-update-stages-indices
+      author: mvulfson
+      changes:
+      - createIndex:
+          indexName: pipeline_stages_updated_at_idx
+          tableName: pipeline_stages
+          columns:
+          - column:
+              name: updated_at
```

Comment:
@asher , just confirming that secondary indices can be non-unique (and if anything else needs to change here - my first migration)